src/main/webapp/app/overview/student-questions/student-questions.scss

```diff
@@ -1,15 +1,108 @@
 @import 'src/main/webapp/content/scss/artemis-variables';
 
-.add-action {
-    color: $primary;
+$background-color: #353d47;
+
+.questions-container {
+    display: flex;
+    height: 100%;
+}
+
+.expanded-questions {
+    width: 30vw;
+
+    .draggable-left {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        min-width: 15px;
+    }
+
+    .card {
+        width: inherit;
+        min-width: 200px;
+
+        .card-header {
+            display: inline-flex;
+            justify-content: space-between;
+            align-items: center;
+            background-color: $background-color;
+            cursor: pointer;
+
+            .card-title {
+                display: flex;
+            }
+        }
```

Comment:
```suggestion
        }
        
        .card-body {
            padding-top: 10px !important;
            padding-bottom: 10px !important;
            > .row >.col-12:first-child > button {
                margin: 0 15px;
            }
        }
```

Reply:
I implemented that in my last commit, but not using the scss but by helper classes